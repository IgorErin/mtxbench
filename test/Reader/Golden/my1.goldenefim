QNode
    { lt = QLeaf { value = Nothing }
    , rt = QLeaf
        { value = Just 3 }
    , ll = QLeaf
        { value = Just 1 }
    , rl = QLeaf
        { value = Just 2 }
    }