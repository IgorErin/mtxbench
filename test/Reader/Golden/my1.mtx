%%MatrixMarket matrix coordinate integer general
11 8 3
1 2 3
2 2 2
2 3 1