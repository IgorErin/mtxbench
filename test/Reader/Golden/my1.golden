QNode
    { lt = QNode
        { lt = QLeaf { value = Nothing }
        , rt = QNode
            { lt = QLeaf { value = Nothing }
            , rt = QLeaf { value = Nothing }
            , ll = QLeaf
                { value = Just 3 }
            , rl = QLeaf { value = Nothing }
            }
        , ll = QLeaf { value = Nothing }
        , rl = QNode
            { lt = QLeaf
                { value = Just 2 }
            , rt = QLeaf
                { value = Just 1 }
            , ll = QLeaf { value = Nothing }
            , rl = QLeaf { value = Nothing }
            }
        }
    , rt = QLeaf { value = Nothing }
    , ll = QLeaf { value = Nothing }
    , rl = QLeaf { value = Nothing }
    }